Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -269,7 +273,7 @@ private func validKeys(ruleList: RuleList) -> [String] {
     ].map({ $0.rawValue }) + ruleList.allValidIdentifiers()
 }
 
-private func warnAboutDeprecations(_ dict: [String: Any],
+private func warnAboutDeprecations(dict: [String: Any],
```

Comment:
I was in doubt on `configuration` vs `configurationDictionary`  because as we have a `Configuration` type I'd expect that a `configuration` parameter would be of that type.